ui/analyse/src/explorer/explorerConfig.ts

```diff
@@ -317,6 +317,16 @@ const playerModal = (ctrl: ExplorerConfigCtrl) => {
     ctrl.selectPlayer(name);
     ctrl.root.redraw();
   };
+  const nameToOptionalColor = (name: string | undefined) => {
+    if (!name) {
+      return;
+    } else if (name === ctrl.myName) {
+      return '.button-green';
+    } else if (ctrl.data.playerName.previous().includes(name)) {
+      return '';
+    }
+    return '.button-metal';
```

Comment:
Why change the colour for game participants? It seems unnecessary.

Reply:
While it would be nice to distinguish between participants whose names are in the search history and those that aren't, using that shade of gray to denote the participants who aren't in the search history can be controversial.

Reply:
(Aesthetically-speaking)

Reply:
Since users cannot add participants to the search history, it makes sense to gray out a participant if they are not in the search history.